rest-tests/src/test/resources/features/operator-deployment.feature

```diff
@@ -0,0 +1,243 @@
+# @sustainer: avano@redhat.com
+
+@operator
```

Comment:
todo: add ignore here so that it is not run by default, it will use a separate job